canvasobject.go

```diff
@@ -96,3 +96,10 @@ type Focusable interface {
 type Shortcutable interface {
 	TypedShortcut(Shortcut)
 }
+
+// Tabable describes any object that needs to accept the Tab key presses.
+type Tabable interface {
```

Comment:
This needs to have the API version added, sorry I missed this before. Add the following to the doc:
```go
\\
\\ Since: 2.1
```